adapters/mongoose.js

```diff
@@ -131,6 +131,10 @@ module.exports = function (model, opts) {
     } else if (opts.instance === 'ObjectID') {
       // Deal with ObjectID
       return 'String';
+    } else if (opts.instance === 'Embedded') {
+      return objectType(opts.schema.obj, function (fieldName) {
+        return getTypeFromNative(opts.schema.obj[fieldName].type);
+      });
```

Comment:
Does not work properly with this code:

```
var bluSchema = new mongoose.Schema({
    a: String,
    b: Boolean,
    c: Date,
    d: Number,
    e: { type: String, enum: ['abababa', 'b', 'c'] }
});

blu1: bluSchema,
blu2: {
    a: String,
    b: Boolean,
    c: Date,
    d: Number,
    e: { type: String, enum: ['abababa', 'b', 'c'] }
  },
```
`blu1` and `blu2` should behave the same. It does not seem to be the same.